pkgs/development/python-modules/duckdb/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, duckdb
+, numpy
+, pandas
+, pybind11
+, setuptools_scm
+, pytestrunner
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "duckdb";
+  version = duckdb.version;
+
+  src = duckdb.src;
```

a little more concise, but I don't feel super strongly about it:
```suggestion
  inherit (duckdb) version src;
```

---

I like that. Shows intent better.

---

resolved.